ci/docker/Dockerfile

```diff
@@ -48,10 +48,29 @@ RUN chmod +x /usr/local/bin/initdocker
 RUN curl -Lo /usr/local/bin/dunit-progress https://github.com/jdeppe-pivotal/progress-util/releases/download/0.2/progress.linux
 RUN chmod +x /usr/local/bin/dunit-progress
 
+# Install Chrome
+RUN apt-get update && apt-get install -y apt-transport-https ca-certificates curl gnupg --no-install-recommends \
```

Comment:
This apt-get update and install shouldn't be necessary because they're installed earlier in the dockerfile.